tools/sentieon_DNAscope_LongRead_CLI.cwl

```diff
@@ -0,0 +1,229 @@
+      required: false
+    - pattern: .bai
+      required: false
+    - pattern: .crai
+      required: false
+  align:
+    type: boolean?
+    default: false
+    inputBinding:
+      position: 5
+      prefix: --align
+  model_bundle:
+    type: File
+    doc: "The model bundle file"
+    inputBinding:
+      position: 6
+      prefix: -m
+  tech:
+    type: string?
+    doc: "{HiFi,ONT}     Sequencing technology used to generate the reads. (default: 'HiFi')"
+    inputBinding:
+      position: 7
+      prefix: --tech
+  dbSNP:
+    type: File?
+    doc: "dbSNP vcf file Supplying this file will annotate variants with their dbSNP refSNP ID numbers."
+    inputBinding:
+      position: 8
+      prefix: -d
+    secondaryFiles:
+    - pattern: .tbi
+      required: false
+    - pattern: .idx
+      required: false
+  diploid_bed:
+    type: File?
+    doc: "Region BED file. Supplying this file will limit variant calling to the intervals inside the BED file."
+    inputBinding:
+      position: 9
+      prefix: -b
+    sbg:fileTypes: BED
+  haploid_bed:
+    type: File?
+    inputBinding:
+      position: 10
+      prefix: --haploid-bed
+    sbg:fileTypes: BED
+  gvcf:
+    type: boolean?
+    default: false
+    inputBinding:
+      position: 11
+      prefix: --gvcf
+  bam_format:
+    type: boolean?
+    default: false
+    doc: "Use the BAM format instead of CRAM for output aligned files (default: False)"
+    inputBinding:
+      position: 12
+      prefix: --bam_format
+  cores:
+    type: int?
+    doc: "Number of threads/processes to use"
+    inputBinding:
+      position: 13
+      prefix: -t
+  skip-small-variants:
+    type: boolean?
+    default: false
+    doc: "Skip small variant (SNV/indel) calling (default: False)"
+    inputBinding:
+      position: 14
+      prefix: --skip-small-variants
+  skip-svs:
+    type: boolean?
+    default: false
+    doc: "Skip SV calling (default: False)"
+    inputBinding:
+      position: 15
+      prefix: --skip-svs
+  skip-mosdepth:
+    type: boolean?
+    default: false
+    doc: "Skip QC with mosdepth (default: False)"
+    inputBinding:
+      position: 16
+      prefix: --skip-mosdepth
+  input_ref:
+    type: File?
+    doc: "Used to decode the input alignment file. Required if the input file is in the CRAM/uCRAM formats"
+    inputBinding:
+      position: 17
+      prefix: --input_ref
+    secondaryFiles:
+    - pattern: .fai
+      required: true
+  fastq_taglist:
+    type: string?
+    doc: "A comma-separated list of tags to retain. Defaults to ''*'' and the 'RG' tag is required"
+    inputBinding:
+      position: 18
+      prefix: --fastq_taglist
+  minimap2_args:
+    type: string?
+    doc: "Extra arguments for sentieon minimap2 (default: '-Y')"
+    inputBinding:
+      position: 20
+      prefix: --minimap2_args
+  util_sort_args:
+    type: string?
+    doc: "Extra arguments for sentieon util sort (default: '--cram_write_options version=3.0,compressor=rans')"
```

Comment:
So --util_sort_args takes in by default an arg that looks like an option? Maybe it's a wrapper thing that's not apparent from here

Reply:
That's what it says. Seems like it's the equivalent of doing "extra args"